app/brave_main_delegate_browsertest.cc

```diff
@@ -91,6 +91,7 @@ IN_PROC_BROWSER_TEST_F(BraveMainDelegateBrowserTest, DisabledFeatures) {
     &autofill::features::kAutofillEnableAccountWalletStorage,
     &autofill::features::kAutofillServerCommunication,
     &blink::features::kAdInterestGroupAPI,
+    &blink::features::kCssSelectorFragmentAnchor,
```

Comment:
sort?

Reply:
Yep, forgot my ABCs .. Fixed in 75ff228aaed2ff61c7f2a182237ad83cca918e80